tests/manage/test_pvc_disruptive.py

```diff
@@ -21,43 +61,171 @@ class BaseDisruption(ManageTest):
-        )
+
+        # Start creation of multiple PVCs
+        bulk_pvc_create = executor.submit(self.create_pvcs)
+
         if operation_to_disrupt == 'create_pvc':
-            DISRUPTION_OPS.delete_resource()
-        self.pvc_obj.reload()
-        assert self.pvc_obj.ocp.wait_for_resource(
-            condition=constants.STATUS_BOUND, resource_name=self.pvc_obj.name, timeout=120
-        )
-
-        self.pod_obj = helpers.create_pod(
-            interface_type=constants.CEPHBLOCKPOOL, pvc_name=self.pvc_obj.name, wait=False
-        )
+            # Ensure PVCs are being created before deleting the resource
+            for count in range(10):
+                current_num_of_pvc = len(get_all_pvcs(namespace=self.namespace)['items'])
+                if current_num_of_pvc > initial_num_of_pvc:
+                    num_of_new_pvc = current_num_of_pvc - initial_num_of_pvc
+                    logging.info(f"PVCs creation has started. Found {num_of_new_pvc} new PVC.")
+                    DISRUPTION_OPS.delete_resource()
+                    break
+                assert count < 9, "PVCs are not being created."
+                logging.info("Creation of PVCs has not started. Checking again.")
+
+        pvc_objs = bulk_pvc_create.result()
+
+        # Verify PVCs are Bound
+        for pvc_obj in pvc_objs:
+            pvc_obj.reload()
+            assert pvc_obj.ocp.wait_for_resource(
+                condition=constants.STATUS_BOUND, resource_name=pvc_obj.name, timeout=120
+            )
+        logging.info("Verified: PVCs are Bound.")
+
+        # Start creating pods
+        bulk_pod_create = executor.submit(self.create_pods, pvc_objs)
+
         if operation_to_disrupt == 'create_pod':
-            DISRUPTION_OPS.delete_resource()
-        self.pod_obj.reload()
+            for count in range(10):
+                # Ensure that pods are being created before deleting the resource
+                current_num_of_pod = len(pod.get_all_pods(namespace=self.namespace))
+                if current_num_of_pod > initial_num_of_pods:
+                    num_of_new_pod = current_num_of_pod - initial_num_of_pods
+                    logging.info(f"Pods creation has started. Found {num_of_new_pod} new pod.")
+                    DISRUPTION_OPS.delete_resource()
+                    break
+                assert count < 9, "PVCs are not being created."
+                logging.info("Creation of Pods has not started. Checking again.")
+
+        pod_objs = bulk_pod_create.result()
+
+        # Verify pods are Running
+        for pod_obj in pod_objs:
+            pod_obj.reload()
+            assert pod_obj.ocp.wait_for_resource(
+                condition=constants.STATUS_RUNNING, resource_name=pod_obj.name, timeout=120
+            )
+        logging.info("Verified: All pods are Running.")
+
+        # Start IO on each pod
+        for pod_obj in pod_objs:
+            pod_obj.run_io(storage_type='fs', size='1G', fio_filename='fio-file1')
```

Comment:
check the runtime.
I think the default is 60 seconds which is too much for 1G

Reply:
It is lesser than 10 seconds. Updated runtime=10